ft_volumewrite.m

```diff
@@ -110,13 +110,15 @@ function ft_volumewrite(cfg, volume)
 cfg.markorigin   = ft_getopt(cfg, 'markorigin',   'no');
 cfg.markfiducial = ft_getopt(cfg, 'markfiducial', 'no');
 cfg.markcorner   = ft_getopt(cfg, 'markcorner',   'no');
-cfg.scaling      = ft_getopt(cfg, 'scaling',      'no');
+cfg.scaling      = ft_getopt(cfg, 'scaling');
 
-if any(strmatch(cfg.datatype, {'int8', 'int16', 'int32'}))
-  cfg.scaling = 'yes';
+if any(strmatch(cfg.datatype, {'uint8','int8', 'int16', 'int32'}))&&isempty(cfg.scaling)
```

Comment:
should we be exhaustive here, and also include uint16/uint32 (u)int64?

Reply:
then you also need to include them around line #200. The rescaling is not yet implemented for these datatypes and they are also not mentioned in the help. If you think this could be of use it is not a big thing implementing them.